link/src/main/java/com/stripe/android/link/ui/wallet/WalletModals.kt

```diff
@@ -0,0 +1,96 @@
+package com.stripe.android.link.ui.wallet
+
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.AlertDialog
+import androidx.compose.material.Text
+import androidx.compose.material.TextButton
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+import com.stripe.android.link.R
+import com.stripe.android.link.theme.HorizontalPadding
+
+@Composable
+internal fun WalletBottomSheetContent(
+    onCancelClick: () -> Unit,
+    onRemoveClick: () -> Unit
+) {
+    Column(
+        modifier = Modifier
+            .fillMaxWidth()
+    ) {
+        Row(
+            modifier = Modifier
+                .fillMaxWidth()
+                .clickable(onClick = onRemoveClick)
+        ) {
+            Text(
+                text = stringResource(R.string.wallet_remove_card),
+                modifier = Modifier
+                    .padding(
+                        start = HorizontalPadding,
+                        top = 24.dp,
+                        end = HorizontalPadding,
+                        bottom = 10.dp
+                    )
+            )
+        }
+
+        Row(
+            modifier = Modifier
+                .fillMaxWidth()
+                .clickable(onClick = onCancelClick)
+        ) {
+            Text(
+                text = stringResource(R.string.cancel),
+                modifier = Modifier
+                    .padding(
+                        start = HorizontalPadding,
+                        top = 10.dp,
+                        end = HorizontalPadding,
+                        bottom = 24.dp
+                    )
+            )
+        }
+    }
+}
+
+@Composable
+internal fun ConfirmRemoveDialog(
+    showDialog: Boolean,
+    onDialogDismissed: (Boolean) -> Unit
+) {
+    if (showDialog) {
+        AlertDialog(
```

Comment:
We have an internal implementation for `AlertDialog` https://github.com/stripe/stripe-android/blob/master/payments-ui-core/src/main/java/com/stripe/android/ui/core/elements/SimpleDialogElementUI.kt

Reply:
I'll look into reusing, but won't do it right now just for the sake of speed. It'll need some refactoring for the styling and not showing a title, for example.